src/app/views/kyb.go

```diff
@@ -137,23 +148,25 @@ func kybVerificationGroup(router *gin.Engine) {
 			return
 		}
 
-		c.JSON(http.StatusOK, gin.H{})
+		c.JSON(http.StatusOK, verification)
 	})
 
 	g.GET("/:id/reject", adminAccessRequired(), func(c *gin.Context) {
 		ctx, _ := c.Get("ctx")
 		verificationId := c.Param("id")
 
-		kybVerification := models.KYBVerification{
-			ID: uuid.MustParse(verificationId),
+		verification, err := models.GetById(uuid.MustParse(verificationId))
+		if err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+			return
 		}
 
-		if err := kybVerification.ChangeStatus(ctx.(context.Context), models.KYBStatusRejected); err != nil {
+		if err := verification.ChangeStatus(ctx.(context.Context), models.KYBStatusRejected); err != nil {
 			c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
 			return
 		}
 
-		c.JSON(http.StatusOK, gin.H{})
+		c.JSON(http.StatusOK, verification)
```

Comment:
no need to response verification on admin operators just {message: 'success'} would be enough
```suggestion
		c.JSON(http.StatusOK, gin.H{"message": "success"})
```